macOS_Catalina.md

```diff
@@ -52,18 +50,23 @@ There are two main ways to install `node-gyp` on macOS:
 10. `sudo xcode-select --reset`  # Enter root password.  No output is normal.
 11. Repeat step 7 above.  Is the path different this time?  Repeat the _acid test_.
 
-### Installing `node-gyp` using the Xcode Command Line Tools
+### Installing `node-gyp` using the Xcode Command Line Tools via `xcode-select --install`
 1. If the _acid test_ has not succeeded, then try `xcode-select --install`
 2. Wait until the install process is _complete_.
 3. `softwareupdate -l`  # No listing is a good sign.
     * If Xcode or Tools upgrades are listed, use "Software Update" to install them.
 4. `xcode-select -version`  # Should return `xcode-select version 2370` or later.
 5. `xcode-select -print-path`  # Should return `/Library/Developer/CommandLineTools`
-6. Try the [_acid test_ steps above](#The-acid-test) to see if your Mac is ready.
+6. Try the [_acid test steps above](#The-acid-test) to see if your Mac is ready.
```

Comment:
This starts _italics_ but never finishes it.

Reply:
Thanks for catching that.